applications/StructuralMechanicsApplication/python_scripts/element_sensitivity_domain_integration_process.py

```diff
@@ -56,7 +56,7 @@ def __init__(self, model, parameter):
 
         # Get defined sub model parts of sensitivity model part as integration domains
         self.sensitivity_sub_model_parts = []
-        if len(sensitivity_sub_model_part_names) is 0:
+        if len(sensitivity_sub_model_part_names) == 0:
```

Comment:
Better remove the == 0 altogether 

Can find the pep for it later

Reply:
https://peps.python.org/pep-0008/

Reply:
Trivia: In Catalan/Valencian, Pep is a short for Joseph (Pepe in Spanish), and every time I see PEP8 I think in a song of my childhood:

https://www.comarcarural.com/valencia/musica/eltiopep.php

Muro is just the closest small village to my hometown